### PR TITLE
Create galileo skill

### PR DESCRIPTION
**Pull request by Rafael-Martins**
# Implement Galileo skills 

Galileo will be a set of commands that will help see things in the sky. At this first pr, we are adding a basic command to predict visual passes from the international space station as shown bellow:

screen shoot

A future upgrade in this will be use a geocoder to pass the city name instead the coordinates!

### New env is needed! `N2YO_API_TOKEN` I'll provide one for the deploy, just text me :package:

### CODE REVIEW

**Comment by Rafael-Martins** [comment]
Missing docs

File: src/skills/galileo/commands/nextIssPasses.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { CommandMiddleware } from "grammy/composer.ts";
+import { BotContext } from "/src/context/mod.ts";
+import { parsePosition } from "../utilities/parsePosition.ts";
+import { GeoPosition } from "./types.ts";
+import { formatIssPassMessage } from "../utilities/formatIssPassMessage.ts";
+
+export const nextIssPasses: CommandMiddleware<BotContext> = async (ctx) => {
+  const position: GeoPosition = parsePosition(ctx.match);
+  const { latitude, longitude } = position;
```
Comment (armand1m) [inline src/skills/galileo/commands/nextIssPasses.ts]:
```suggestion
  const { latitude, longitude } = parsePosition(ctx.match);
```

File: src/skills/galileo/utilities/parsePosition.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const parsePosition = (text: string) => {
```
Comment (armand1m) [inline src/skills/galileo/utilities/parsePosition.ts]:
```suggestion
import { GeoPosition } from "/src/skills/galileo/commands/types.ts";
export const parsePosition = (text: string): GeoPosition => {
```

**Review by armand1m — CHANGES_REQUESTED** [review]
So far everything looking good. Added a few suggestions. Please add the feature details to the README.md for users to be aware of the feature.

File: README.md [hunk 3]
```diff
@@ -125,11 +125,11 @@ code with references and function names like `getTretas(fromThisPessoa)`.
...
 ### Hashtags
 
-- [ ] `/join_hashtag <hashtag>`: user joins to the selected channel and will be
+- [x] `/join_hashtag <hashtag>`: user joins to the selected channel and will be
       mentioned every time that hashted is used. Example: `/join_hashtag #games`
-- [ ] `/leave_hashtag #subject`: user leave the channel and will be not
+- [x] `/leave_hashtag #subject`: user leave the channel and will be not
       mentioned anymore.
-- [ ] Listens to hashtags in the chat and mentions people registered to those
+- [x] Listens to hashtags in the chat and mentions people registered to those
```
Comment (armand1m) [inline README.md:132]: thanks for this!

**Review by armand1m — APPROVED** [review]

**Comment by armand1m** [comment]
I approved but the feature will only work if the secret is set in fly io. I'm afk but you have permissions to do that @Rafael-Martins

**Pull request merged**